### DISCUSSION TITLE
'wslink.launcher' not found after building trame image with paraview

### DISCUSSION THREAD

**Discussion by smkacur** [Kitware/trame#643] [Trame/ParaView] [answered]
Hi @Jourdain,
I am using the dockerfile below to build trame and deploy using docker compose. I was successfully using this file a few months ago but when I tried to use it this morning I get this error: 
` /deploy/server/venv/bin/python: Error while finding module specification for 'wslink.launcher' (ModuleNotFoundError: No module named 'wslink')`

Here is the dockerfile:
```python
FROM --platform=linux/amd64 kitware/trame:py3.9 as trame

RUN install -d -o trame-user -g trame-user /deploy
RUN apt update -y && apt install -y libosmesa6-dev
ARG PV_URL='https://www.paraview.org/files/v5.11/ParaView-5.11.0-osmesa-MPI-Linux-Python3.9-x86_64.tar.gz'
RUN mkdir -p /opt/paraview && cd /opt/paraview && wget -qO- $PV_URL | tar --strip-components=1 -xzv
ENV TRAME_PARAVIEW=/opt/paraview
ENV TRAME_CLIENT_TYPE=vue3

COPY --chown=trame-user:trame-user . /deploy

RUN /opt/trame/entrypoint.sh build
```

I saw this issue looks related https://github.com/Kitware/trame/discussions/534 but thought that having `RUN /opt/trame/entrypoint.sh build` at the end of the dockerfile would handle building the server directory within the container.

**Comment by jourdain** [comment, marked as answer]
yes it would, but it is failing in your case. Looking at the docker build log will help you identify why it is failing on your end.

**Reply by smkacur** [reply]
Thank you! I had a typo in the requirements.txt that was getting installed during the server directory build.
